config.example.js

```diff
@@ -113,6 +113,9 @@ var config = {
       feeds : [],  // RSS feeds list - e.g. ["rss1.com", "rss2.com"]
       refreshInterval : 120 // Number of minutes the information is refreshed
     },
+    stock: {
+      names: '' // The names of the stock quotes you with to show in the official format. (e.g.: YHOO,AAPL,GOOG)
```

Comment:
Shouldn't this be an array? It would make a bit more sense for someone filling out the config and would likeley be less error prone.